frontend/packages/kubevirt-plugin/src/components/vm-templates/VMTemplateSupport.tsx

```diff
@@ -4,12 +4,13 @@ import { useTranslation } from 'react-i18next';
 import { ExternalLink } from '@console/internal/components/utils';
 
 import { SUPPORT_URL } from '../../constants/vm-templates';
+import { isUpstream } from '../../utils/common';
 
 const VMTemplateSupport: React.FC<VMTemplateSupportProps> = ({ details }) => {
   const { t } = useTranslation();
-  const isUpstream = window.SERVER_FLAGS.branding === 'okd';
+  const upstream = isUpstream();
```

Comment:
nitpick: Couldn't we just call it inline in all of the places it's currently used since none of them use a loop? I think it would read a little better this way in addition to declaring another variable.